src/tasks/dev_prerun.js

```diff
@@ -38,9 +38,24 @@ async function configureKycContract(dataModelEngine, logger) {
     });
 }
 
+async function onboardAsHermes(dataModelEngine, logger) {
+  logger.info(`Onboarding node as HERMES`);
+  const address = dataModelEngine.identityManager.nodeAddress();
+  const rolesContract = await dataModelEngine.contractManager.rolesContract();
+  const url = `localhost:${config.serverPort}`;
+  await rolesContract
+    .methods
+    .onboardAsHermes(url)
+    .send({
+      from: address,
+      gas: 2000000
+    });
+}
+
 async function setupDevelopment(dataModelEngine, logger) {
   await registerAdminAccount(dataModelEngine, logger);
   await configureKycContract(dataModelEngine, logger);
+  await onboardAsHermes(dataModelEngine, logger);
```

Comment:
Maybe move adding to kyc whitelist to onboarding method, as this actually is part of the onboarding process?
If not then maybe rename `configureKycContract` --> `addToKycWhitelist` or sth.